<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#widget_2 {
	border: 2px solid #5C5C5C;
	background-color: #333333;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_7 {
	border-bottom: 2px solid #5C5C5C;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"> background-color: #1E1E2F; /* xanh navy đậm */
    color: #F5F5F5; /* chữ sáng dễ đọc */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::AutoText</enum>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/new/image/image/Picture1.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"> background-color: #1E1E2F; /* xanh navy đậm */
    color: #F5F5F5; /* chữ sáng dễ đọc */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <widget class="QLabel" name="label_TokenName">
             <property name="geometry">
              <rect>
               <x>-10</x>
               <y>10</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Token</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogin">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
            </property>
            <property name="text">
             <string>Chọn Token</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectSignatureImage">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
            </property>
            <property name="text">
             <string>Chọn chữ ký</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectPos">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
            </property>
            <property name="text">
             <string>Chọn vị trí</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectDocument">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
            </property>
            <property name="text">
             <string>Chọn văn bản</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectTokenAgain">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
            </property>
            <property name="text">
             <string>Chọn lại thiết bị</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #333333;
border-top: 2px solid #5C5C5C;
border-bottom: 2px solid #5C5C5C;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ClickableLabel" name="label_2">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/new/image/image/dropFileHere.jpg</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #1E1E2F; /* xanh navy đậm */
    color: #F5F5F5; /* chữ sáng dễ đọc */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #1E1E2F; /* xanh navy đậm */
    color: #F5F5F5; /* chữ sáng dễ đọc */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;</string>
            </property>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>10</y>
               <width>41</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Trang:</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnBackFisrtPage">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>7</y>
               <width>31</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>&lt;&lt;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnBackPage">
             <property name="geometry">
              <rect>
               <x>155</x>
               <y>7</y>
               <width>31</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEditPageNum">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>7</y>
               <width>50</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="btnNextLastPage">
             <property name="geometry">
              <rect>
               <x>355</x>
               <y>7</y>
               <width>31</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>&gt;&gt;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnNextPage">
             <property name="geometry">
              <rect>
               <x>320</x>
               <y>7</y>
               <width>31</width>
               <height>24</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
            <widget class="QLabel" name="labelTotalPageNum">
             <property name="geometry">
              <rect>
               <x>255</x>
               <y>9</y>
               <width>71</width>
               <height>20</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>999999</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_saperatorPageNum">
             <property name="geometry">
              <rect>
               <x>245</x>
               <y>11</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>/</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>100</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>15</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btnAccept">
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
                  </property>
                  <property name="text">
                   <string>Xác nhận</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnReSign">
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    background-color: #2C2C3A;
    border: 1px solid #444;
    border-radius: 6px;
    padding: 6px 12px;
    color: #FFFFFF;
}

QPushButton:hover {
    background-color: #3A3A4D;
}

</string>
                  </property>
                  <property name="text">
                   <string>Ký lại</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnCancel">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>-1</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
QPushButton {
         background-color: #ffe6f0;    
        border: 1px solid #ff99cc;   
        border-radius: 4px;         
        color: black;                  
    }
    QPushButton:hover {
        background-color: red;  
        color: white;          
    }
    QPushButton:pressed {
        background-color: darkred; 
        color: yellow;          
    }</string>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAbout_me">
   <property name="text">
    <string>About me</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
